src/plugins/workspace/public/components/workspace_list/index.tsx

```diff
@@ -320,28 +320,27 @@ export const WorkspaceListInner = ({
       setSelection([]);
     };
 
-    return (
-      isDashboardAdmin && (
-        <>
-          <EuiButton color="danger" iconType="trash" onClick={onClick}>
-            Delete {selection.length} Workspace
-          </EuiButton>
-          {deletedWorkspaces && deletedWorkspaces.length > 0 && (
-            <DeleteWorkspaceModal
-              selectedWorkspaces={deletedWorkspaces}
-              onClose={() => setDeletedWorkspaces([])}
-            />
-          )}
-        </>
-      )
-    );
+    return isDashboardAdmin ? (
+      <>
+        <EuiButton color="danger" iconType="trash" onClick={onClick}>
+          Delete {selection.length} {selection.length > 1 ? 'workspaces' : 'workspace'}
```

Comment:
Let's add i18n for the text, take this as example:
```typescript
<FormattedMessage
  id="dataSourcesManagement.dataSourcesTable.dissociateSelectedDataSources"
  defaultMessage="{selectionSize, plural, one {Remove # association} other {Remove # associations}}"
  values={{ selectionSize: selectedDataSources.length }}
/>
```

Reply:
Thanks for your comments, already updated.